src/api/application/application-config.ts

```diff
@@ -42,7 +42,7 @@ export const applicationConfig = {
   DECLARATION_AUDIT_LOCATIONS: 'DISTRICT',
   FEATURES: {
     DEATH_REGISTRATION: true,
-    MARRIAGE_REGISTRATION: false,
```

Comment:
Is this intentional?

Reply:
The certificateTemplate feature has it's implementation in Marriage events too. So, in the feature branch deployment, I wanted to turn on the Marriage so that QAs can test.

Reply:
@tareq89 We should keep marriage disabled in country config